hypervisor/arch/x86/guest/vm_reset.c

```diff
@@ -82,29 +82,37 @@ static bool handle_common_reset_reg_write(struct acrn_vm *vm, bool reset)
 {
 	bool ret = true;
 
-	if (is_highest_severity_vm(vm)) {
-		if (reset) {
+	if (reset) {
+		if (is_highest_severity_vm(vm)) {
 			reset_host();
-		}
-	} else if (is_postlaunched_vm(vm)) {
-		/* re-inject to DM */
-		ret = false;
+		} else if (is_postlaunched_vm(vm)) {
+			/* re-inject to DM */
+			ret = false;
 
-		if (reset && is_rt_vm(vm)) {
-			vm->state = VM_POWERING_OFF;
-		}
-	} else if (is_prelaunched_vm(vm)) {
-		/* Don't support re-launch for now, just shutdown the guest */
-		pause_vm(vm);
+			if (is_rt_vm(vm)) {
+				vm->state = VM_POWERING_OFF;
+			}
+		} else {
+			/*
+			 * If it's SOS reset while RTVM is still alive
```

Comment:
How to align with the code "While RTVM is still alive"?

Reply:
OK, got it, because it current is not the hightest severity_vm. Please ignore this comment.